pkg/crypto/certificatemanager/certificate_manager.go

```diff
@@ -51,33 +54,59 @@ func (mc managerConfig) Flags(flags *pflag.FlagSet) {
 // Manager will manage the way certificates are retrieved based in the given
 // k8sClient and rootPath.
 type manager struct {
-	rootPath  string
-	k8sClient k8sClient.Clientset
+	rootPath            string
+	k8sClient           k8sClient.Clientset
+	secretSyncNamespace string
+	Logger              logrus.FieldLogger
 }
 
 // NewManager returns a new manager.
-func NewManager(cfg managerConfig, clientset k8sClient.Clientset) (CertificateManager, SecretManager) {
+func NewManager(cfg managerConfig, clientset k8sClient.Clientset, logger logrus.FieldLogger) (CertificateManager, SecretManager) {
 	m := &manager{
 		rootPath:  cfg.CertificatesDirectory,
 		k8sClient: clientset,
+		Logger:    logger,
 	}
 
+	m.Logger.Debug("certificate-manager starting")
```

Comment:
is this log message a leftover from a debug session? 

just mentioning it because it's misleading having this "starting" in the Hive config lifecycle (where the component isn't actually started - just configured). 

Therefore, I'd recommend to either remove the log message (if no longer necessary) or rephrase it (`Initializing certificate-manager`).

Reply:
Removed.